trace/tracer.go

```diff
@@ -36,6 +33,9 @@ type Config struct {
 	// If >= 1 always samples.
 	// If <= 0 never samples.
 	TraceSampleRate float64 `json:"traceSampleRate"`
+
+	AppName string
+	Version string
```

Comment:
Can you add JSON tags for these?

Reply:
Done.
We need to marshal the config at some point?

Reply:
It can be marshaled in the admin API's `GetConfig` method